SlicerProstate/SlicerProstateUtils/mixins.py

```diff
@@ -246,13 +246,13 @@ def get3DEuclideanDistance(pos1, pos2):
     return distance3D
 
   @staticmethod
-  def dilateMask(label):
+  def dilateMask(label, dilateValue, erodeValue=0):
     imagedata = label.GetImageData()
     dilateErode = vtk.vtkImageDilateErode3D()
     dilateErode.SetInputData(imagedata)
-    dilateErode.SetDilateValue(1.0)
-    dilateErode.SetErodeValue(0.0)
-    dilateErode.SetKernelSize(12, 12, 1)
+    dilateErode.SetDilateValue(dilateValue)
+    dilateErode.SetErodeValue(erodeValue)
+    dilateErode.SetKernelSize(3, 3, 3)
```

Comment:
I would think kernelSize should be as it is right now, and be configurable via a parameter too.

If I understand the operation correctly, kernel in mm was 12*.5 x 12*.5 x 1*4 = 6x6x4, and with your proposed change it will be 1.5x1.5x12 mm!